mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -180,6 +183,9 @@ public void onDrawerClosed(View drawerView) {
             @Override
             public void onDrawerOpened(View drawerView) {
                 super.onDrawerOpened(drawerView);
+                Context context = getApplicationContext();
+                InputMethodManager inputMethodManager = (InputMethodManager) context.getSystemService(Activity.INPUT_METHOD_SERVICE);
+                inputMethodManager.hideSoftInputFromWindow(drawerView.getWindowToken() , 0);
```

Comment:
We already have defined a method called `hideKeyboard` in MifosBaseActivity, use that method instead.